modules/oe_theme_content_project/src/Plugin/ExtraField/Display/PercentageExtraField.php

```diff
@@ -106,26 +106,31 @@ public function viewElements(ContentEntityInterface $entity) {
   }
 
   /**
-   * Gets the percentage of total, without decimals.
+   * Gets the percentage of total.
    *
-   * If input values are not greater that 0, returns 0.
+   * If input values are not greater than 0, returns 0.
    *
    * @param float $total
    *   The total value.
    * @param float $part
-   *   The percentage value.
+   *   The contribution part of the total value.
    *
-   * @return float
+   * @return string
    *   Percentage value.
    */
-  protected function getPercentage(float $total, float $part): float {
+  protected function getPercentage(float $total, float $part): string {
```

Comment:
We should keep the float type.

Reply:
Updating this as I suggested using `round()` method the returned value will be a float.